test/DotNetWorkerTests/GrpcWorkerTests.cs

```diff
@@ -130,6 +131,44 @@ public void InitRequest_ReturnsExpectedMetadata()
             Assert.Contains(response.WorkerMetadata.CustomProperties,
                 kvp => string.Equals(kvp.Key, "Worker.Grpc.Version", StringComparison.OrdinalIgnoreCase)
                 && string.Equals(kvp.Value, grpcWorkerVersion, StringComparison.OrdinalIgnoreCase));
+            
+        }
+        
+        [Theory]
+        [InlineData("IncludeEmptyEntriesInMessagePayload", true, "IncludeEmptyEntriesInMessagePayload", true, "True")]
+        [InlineData("IncludeEmptyEntriesInMessagePayload", false, "IncludeEmptyEntriesInMessagePayload", false)]
+        public void InitRequest_ReturnsExpectedCapabilitiesBasedOnWorkerOptions(
```

Comment:
nit - could space out the name to make easier to read "InitRequest_ReturnsExpectedCapabilities_BasedOnWorkerOptions"